developer-docs/latest/plugins/upload.md

```diff
@@ -26,6 +26,30 @@ module.exports = {
 };
 ```
 
+#### Responsive Images
+
+When the `Enable responsive friendly upload` setting is enabled in settings panel the plugin will generate the following responsive image sizes:
+| Name    | Largest Dimension |
+| :------ | :--------- |
+| large   | 1000px     |
+| medium  | 750px      |
+| small   | 500px      |
+
+These sizes can be overridden in `config/plugins.js`:
+```javascript
+module.exports = {
+  upload: {
+    breakpoints: {
+      xlarge: 1920,
+      large: 1000,
+      medium: 750,
+      small: 500,
+      xsmall: 64
+    }
+  }
+}
+```
+
```

Comment:
Can you add a disclaimer on the fact that there won't be any migration if the developer changes the dimensions multiple times.

Reply:
Done